ts/src/main/scala/ch/epfl/ts/brokers/StandardBroker.scala

```diff
@@ -80,6 +80,7 @@ class StandardBroker extends Component with ActorLogging {
       })
     }
 
+    //TODO(sygi): refactor
```

Comment:
Generally done, it would be best to actually have super class for ExecutedOrders, but something wasn't working there.